css/footer.styl

```diff
@@ -0,0 +1,264 @@
+.i-arrow-forward
```

Comment:
please remove this file we have it on our code.

Reply:
remove this file please !

Reply:
are you suggesting renaming this file?

Reply:
no deleting